optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/ConstraintCollectors.java

```diff
@@ -526,36 +526,380 @@ private static Runnable innerCountDistinctLong(CountDistinctLongResultContainer
     // min
     // ************************************************************************
 
+    /**
+     * Returns a collector that finds a minimum value in a group of {@link Comparable} elements.
+     * <p>
+     *     It is expected that the {@link Comparable}'s {@link Comparable#compareTo(Object)} is consistent with
```

Comment:
Consistency: other javadocs don't ident after the `<p>`. Note that we also never do `</p>` (that didn't work well). All in all, I am happy to re-standarize all code on whatever openJDK is doing, but that's for a separate PR. Let's have this PR follow the rest of the code.